app/src/main/java/com/asfoundation/wallet/ui/balance/RestoreWalletPasswordFragment.kt

```diff
@@ -40,7 +40,7 @@ class RestoreWalletPasswordFragment : DaggerFragment(), RestoreWalletPasswordVie
     super.onCreate(savedInstanceState)
     presenter =
         RestoreWalletPasswordPresenter(this, activityView, restoreWalletPasswordInteractor,
-            walletEventSender, CompositeDisposable(), AndroidSchedulers.mainThread(),
+            walletsEventSender, CompositeDisposable(), AndroidSchedulers.mainThread(),
             Schedulers.io(),
             Schedulers.computation())
```

Comment:
Codestyle issue: Try to avoid parameter split and keep as much of them in single line